src/fondant/core/component_spec.py

```diff
@@ -208,7 +197,7 @@ def args(self) -> t.Mapping[str, Argument]:
                 name: Argument(
                     name=name,
                     description=arg_info["description"],
-                    type=arg_info["type"],
+                    type=pydoc.locate(arg_info["type"]),  # type: ignore
```

Comment:
Is it normal I find no docs on pydoc.locate ?

Reply:
Doesn't seem well documented indeed. Got it from [here](https://stackoverflow.com/a/29831586/4098821).